hydra/app/controllers/catalog_controller.rb

```diff
@@ -207,16 +211,20 @@ def index
     end
 
     super
+    fetch_image_files
   end
 
-  # def show
-  #   super
-  #   @metadata = []
-  #   # loop over each field and add it to the metadata
-  #   @document.each_pair do |k,v|
-  #     @metadata << [k, v] unless v.present?
-  #   end
-  # end
+  def show
+    super
+    # @metadata = []
```

Comment:
What's the purpose of carrying these code comments forward?  Is that the original `super` method?

Also see above about method signature (e.g. returning value)

Reply:
code comment is not ours, i'm not removing it just in case they still want to reference it